### ISSUE TITLE
About hyperparameter settings

### ISSUE THREAD

**Issue by dxbdxx** [AbductiveLearning/ABLSim#3]
Hi, I am wondering how to reproduce the performance on 2ADD CIFAR dataset? The  code `python main_1_2.py --dataset 2ADD --images CIFAR` only report 26.27% accuracy.

**Comment by Tony-HYX** [comment]
Hi, I have run the code with the same command (default hyper-parameters) and got the expected result. I guess this is because different versions of pytorch/cupy behave differently. The following is my command to create the corresponding anaconda environment. Maybe you can try to use it to install the packages. Please tell me if the problem still exists.

(cudatoolkit=10.1)
```
conda create -n ablsim python=3.7 -y
conda activate ablsim
conda install -c conda-forge cupy cudatoolkit=10.1 -y
conda install pytorch==1.7.0 torchvision==0.8.0 torchaudio==0.7.0 cudatoolkit=10.1 -c pytorch -y
pip install tqdm opencv-python scikit-learn matplotlib
python main_1_2.py --dataset 2ADD --images CIFAR
```
(cudatoolkit=11.3, for new GPUs)
```
conda create -n ablsim python=3.7 -y
conda activate ablsim
conda install -c conda-forge cupy cudatoolkit=11.3 -y
conda install pytorch==1.12.1 torchvision==0.13.1 torchaudio==0.12.1 cudatoolkit=11.3 -c pytorch -y
pip install tqdm opencv-python scikit-learn matplotlib
python main_1_2.py --dataset 2ADD --images CIFAR
```

**Comment by dxbdxx** [comment]
Thank you for your reply. I reinstall the environment and it works!
